src/main/java/com/simanavets/tenderflexbackend/mapper/UserCreateDtoMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.simanavets.tenderflexbackend.mapper;
+
+import com.simanavets.tenderflexbackend.dto.UserCreateDto;
+import com.simanavets.tenderflexbackend.entity.User;
+import org.springframework.stereotype.Component;
+
+import java.util.function.Function;
+
+@Component
+public class UserCreateDtoMapper implements Function<UserCreateDto, User> {
```

Comment:
Seems there is no reason for using Function interface. You already defined generic types for it, so it will always be the same. It looks like too difficult for such simple method. My point is to make it easier, or maybe you will need it for future - then leave as it is